### DISCUSSION TITLE
[Suggestion] Increase QuickJS's default max stack size from 256KB to something like 4MB

### DISCUSSION THREAD

**Discussion by CreepGin** [Tencent/puerts#1881] [General]
256KB is abysmally low. Increasing it to something between 4MB to 10MB is more reasonable and prevents QuickJS users from encountering stack overflow errors too quickly.

Just something to consider.

**Comment by chexiongsheng** [comment]
I think it's reasonable for JS itself to use 256KB, but due to some wrong in backend-quickjs implementation, haven't called JS_UpdateStackTop to update the stack information. As a result, QuickJS's stack top continues to record the stack top at the time of the virtual machine's construction. If there are many calls made before invoking JS, it can easily lead to a stackoverflow.
I will fix it later.

**Reply by CreepGin** [reply]
Ah gotcha! Thx, appreciate it.

**Comment by chexiongsheng** [comment]
Update quickjs backend to this version: https://github.com/puerts/backend-quickjs/releases/download/QJS_241030/qjs_v8_bin.tgz .
